### ISSUE TITLE
Feature Request: Interactive Video: Quizzes

### ISSUE THREAD

**Issue by connecteev** [vidstack/player#916]
Allow interactive video quiz questions in the middle of a video (at specified timestamps), with the options to:

1. View Quiz Feedback in 2 modes:
   - Mode 1 (Immediate Feedback): Allow User to see whether their answer was correct or incorrect immediately after answering a quiz question
   - Mode 2 (Delayed Feedback): Allow User to see whether their answers were correct or incorrect only after answering all quiz questions
2. Whether or not to Allow User to view their quiz overall score, correct and incorrect answers, and choose whether to retake quiz before proceeding
3. Whether or not to Block user from proceeding until they get ALL the quiz answers correct

All of these options / functionality can be enabled or disabled by the website owner using code / parameters.

For reference, see: https://vimeo.com/blog/post/video-quiz/#what-is-a-video-quiz
And demos like https://liqvidjs.org/

**Comment by mihar-22** [comment]
This is out of scope for us right now. I expect features like this to be implemented on the application end and not in our library. We've built the player to support any markup so you include whatever you like inside.
